airflow/www/static/js/dag/grid/dagRuns/Bar.tsx

```diff
@@ -110,7 +110,11 @@ const DagRunBar = ({
             width="10px"
             height={`${(duration / max) * BAR_HEIGHT}px`}
             minHeight="14px"
-            backgroundColor={stateColors[state]}
+            background={
+              run.notes != null && run.notes !== ''
+                ? `linear-gradient(-135deg,#000 8px, ${stateColors[state]} 0);`
```

Comment:
Could you provide more screenshots of what a dag run / task instance with a note looks like?
I'm not sure if we should have a visual indication on the grid if something has a note.

Reply:
I think we should have a visual indication (as otherwise in a list of 100 TaskInstances, you'd never find the note).
Anyways, I think we can still tune it a bit. I don't have great designer skills, but thought this looks reasonable and is very distinctive. We can also change the color, figured black would just fit easily on top of all the bright colors we have.
<img width="1338" alt="image" src="https://user-images.githubusercontent.com/20257392/192553634-7ade2212-ae12-4b91-95d6-d761b0f8903c.png">

Reply:
Good pointer. I thought that was only for non-API endpoints but I see it is now.
Changing this now.

Reply:
Also added support for this in the Mapped instances. Don't mind the Chrome black inversion :)
<img width="775" alt="image" src="https://user-images.githubusercontent.com/20257392/192575928-d3bec207-cdfc-4e61-83f3-9008141679a9.png">

Reply:
You're right, there's probably a reason for a note. Idk about the black gradient, but I'll think more on what else could work.